admin/test/features/edit_check_window.feature

```diff
@@ -0,0 +1,22 @@
+
+Feature: Edit Check Window
+
+
+  Background:
+    Given I am logged in with a test developer
+    And I am on the manage check windows page
+    And I want to edit a previously added check
```

Comment:
Can we summarize these 3 steps in to one  ? 'Given I want to edit a previously added check' maybe?

Reply:
Done